Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml

```diff
@@ -125,7 +125,7 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-core</artifactId>
-      <version>${dependency.spring.framework.version}</version>
+      <version>${spring.version}</version>
```

Comment:
Isn't this dep on the parent dep management?

Reply:
yes, agree